internal/pkg/show/config/config.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+
+	"github.com/devstream-io/devstream/internal/pkg/show/config/plugin"
+)
+
+var pluginDefaultConfigs = map[string]string{
+	"argocd": plugin.ArgocdDefaultConfig,
+}
+
+func Show() error {
+	plugin := viper.GetString("plugin")
+	if config, ok := pluginDefaultConfigs[plugin]; ok {
+		fmt.Println(config)
+		return nil
+	}
+	return fmt.Errorf("plugin name error: < %s >", plugin)
```

Comment:
If we run `dtm show config` without plugin name, it will print a log like "plugin name error: <  >". This is not user-friendly. We should see if the "plugin" is empty or not, then give useful error messages. For example:

"Empty plugin name. Maybe you forgot to add --plugin=PLUGIN_NAME?"

Reply:
got it